src/apps/competitions/consumers.py

```diff
@@ -39,16 +37,13 @@ async def receive(self, text_data=None, bytes_data=None):
         if sub.phase.hide_output and not sub.phase.competition.user_has_admin_permission(user_pk):
             return
 
-        submission_output_path = os.path.join(settings.TEMP_SUBMISSION_STORAGE, f"{submission_id}.txt")
-        os.makedirs(os.path.dirname(submission_output_path), exist_ok=True)
         data = json.loads(text_data)
         if data['kind'] == 'detailed_result_update':
             data['result_url'] = make_url_sassy(Submission.objects.get(id=submission_id).detailed_result.name)
             # update text data to include the newly added sas url for retrieval on page refresh
             text_data = json.dumps(data)
 
-        async with aiofiles.open(submission_output_path, 'a+') as f:
-            await f.write(f'{text_data}\n')
+        cache.set(f'submission-{submission_id}-log', f'{text_data}\n', 3600 * 25)
```

Comment:
what this replaces was appending to a file I believe, can we append to the cache?

Painfully, after looking at the django redis libs, we may have to switch to jazzband to get raw redis client support?
https://github.com/jazzband/django-redis#raw-client-access

we should set it up to also nicely use connection pools like how we have the current library:
https://github.com/jazzband/django-redis#connection-pools

aaaand probably compress data:
https://github.com/jazzband/django-redis#compression-support

maybe it'd make sense to store this data as a list of lines and RPUSH?
https://stackoverflow.com/a/41374478/1359986